src/pages/welcome.js

```diff
@@ -21,7 +21,8 @@ export const welcomeScreen = async (auth, route) => {
 }
 
 const clinicalOnlySiteArray = ['KPNW', 'KPCO', 'KPHI', 'KPGA'];
-const researchOnlySiteArray = ['MFC', 'UCM'];
+// Allow UCM access to both dashboards in non-prod environments (949)
+const researchOnlySiteArray = location.host === urls.prod ? ['MFC', 'UCM'] : ['MFC'];
```

Comment:
`Grants UCM access to clinical dashboard in dev and stage,` => Is there any confusion?

Reply:
Sorry, how so? We don't have any non-prod environments but dev and stage, right?

Reply:
i'm sorry if making confusion. But as my understanding i think it should be `location.host === urls.prod ? ['MFC'] : ['MFC', 'UCM']`

Reply:
Ah, no. Basically, the researchOnlySiteArray indicates which sites only have the research dashboard (vs. both research and clinical.) UCM only has the research dashboard in prod (when location.host === urls.prod) and now has both dashboards otherwise (the : case in this ternary.)